frontend/dashboard/components/RepoList/ActionLinks.test.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { ActionLinks } from './ActionLinks';
+import { repository } from 'app-shared/mocks/mocks';
+import { textMock } from '../../../testing/mocks/i18nMock';
+
+jest.mock('../MakeCopyModal', () => {
+  return {
+    MakeCopyModal: React.forwardRef(function MockedMakeCopyModal(
+      props,
+      ref: React.Ref<HTMLDivElement>,
+    ) {
+      return <div ref={ref}>Mocked MakeCopyModal</div>;
+    }),
+  };
+});
```

Comment:
Is it necessary to mock this whole MakeCopyModal? It looks so simple

Reply:
I tried using the actual modal in the test, but it has many dependencies which has to be mocked. In the end we end up testing the actual modal itself (which already has its own test)